src/__tests__/01-connection.ts

```diff
@@ -20,10 +21,26 @@ it('I can connect to bitcoind', async () => {
 	expect(chain).toEqual('regtest')
 })
 
-it('I can connect to bank lnd', async () => {
-	const { public_key } = await getWalletInfo({ lnd: lndMain })
-	expect(public_key.length).toBe(64 + 2)
-})
+const lnds = [lnd1, lnd2, lndonchain]
+for (let item in lnds) {
+  it(`I can connect to lnd index ${item}`, async () => {
+    const { public_key } = await getWalletInfo({ lnd: lnds[item] })
```

Comment:
Are `item` and `lnds[item]` not same?

Reply:
item is an index. ie: `item = 0`, then `item = 1`, etc

I'm using `let item in lnds` instead of `let item of lnds` because I want to show the index if we can't connect to some lnd